indico/modules/events/papers/templates/_contributions.html

```diff
@@ -1,10 +1,11 @@
 {% macro render_paper_section(contribution, show_paper=True) %}
+    {% set can_access = (contribution.abstract and contribution.abstract.submitter == session.user) or contribution.is_user_associated(session.user) %}
```

Comment:
But `paper` is None :/

However, `Paper.can_submit()` could use the new `is_user_associated` method.

BTW, we could add a `check_abstract` kwarg to that method and then move the abstract submitter check in there.